### DISCUSSION TITLE
Foktech AC600 Nano 11AC Adapter WORKING!

### DISCUSSION THREAD

**Discussion by urbanedge** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#226] [Show and tell]
Amazing, brilliant app and instructions, thank you!

Fixed my broken internal WiFi card on my 2013 MacBook Pro running Big Sur.

Bought The Fokteck Nano USB adapter off Amazon and works great.

https://www.amazon.co.uk/gp/product/B06XZ5B5G9/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1

Thank you!

**Comment by chris1111** [comment]
Thanks I add Foktech AC600 Nano to the[ working list](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/commit/f7fc943516240f838bcb37c5c146f094c2c743ef)

**Reply by muhamien** [reply]
Hi, I had usb wifi adapter AC600, it not detect when i plugin the usb on my macOS Big Sur.
When i use it in windows detected as Realtek 8811CU

**Comment by chris1111** [comment]
@muhamien 
8811CU not support
